graph/src/data/store/mod.rs

```diff
@@ -910,6 +910,29 @@ impl Entity {
         Id::try_from(self.get("id").unwrap().clone()).expect("the id is set to a valid value")
     }
 
+    /// Return the VID of this entity and if its missing or of a type different than
+    /// i64 it panics.
+    pub fn vid(&self) -> i64 {
+        self.get("vid")
```

Comment:
It would be better to have a constant `VID: &str = "vid"` to guard against typos. There might even be one defined somewhere already